### DISCUSSION TITLE
Can you make the now bot ignore specific branches?

### DISCUSSION THREAD

**Discussion by Andre601** [vercel/vercel#3994] [General]
I use a GitHub Action to automatically push changes on the `master` branch towards the `gh-pages` branch and want the now bot to ignore those two branches.
The issue I face is, that the bot automatically comments on all commits I make to either of those branches and is just plain annoying in that regard, as I only want it to deploy changes made in PRs.

I saw that there was [an issue](https://github.com/zeit/now/issues/3166) requesting such a feature, but I would like to know now, if this is still a planned feature, or perhaps already available to use.

**Comment by Vadorequest** [comment]
Take a look at https://github.com/UnlyEd/next-right-now/tree/v1-ssr-mst-aptd-gcms-lcz-sty/.github/workflows we use this to deploy depending on which branch it's pushed to :)
